framework/console/Request.php

```diff
@@ -66,11 +66,14 @@ public function resolve()
 
         $params = [];
         foreach ($rawParams as $param) {
-            if (preg_match('/^--(\w+)(=(.*))?$/', $param, $matches)) {
+            if (preg_match('/^--(\w+)(?:=(.*))?$/', $param, $matches)) {
                 $name = $matches[1];
                 if ($name !== Application::OPTION_APPCONFIG) {
-                    $params[$name] = isset($matches[3]) ? $matches[3] : true;
+                    $params[$name] = isset($matches[2]) ? $matches[2] : true;
                 }
+            } elseif (preg_match('/^-(\w+)(?:=(.*))?$/', $param, $matches)) {
+                $name = $matches[1];
+                $params['alias'][$name] = isset($matches[2]) ? $matches[2] : true;
```

Comment:
just noticed, that it is not possible to have an option named `alias` after this change. the array key should be something that does not match `(\w+)`. maybe `_aliases`?